cmd/cmdutils.go

```diff
@@ -170,3 +172,30 @@ func printUnmountedStorage(client *occlient.Client, applicationName string) {
 	}
 	fmt.Println("")
 }
+
+// GetAppName returns application name
+func GetAppName(client *occlient.Client) string {
+	project := project.GetCurrent(client)
```

Comment:
Should we also consider the `Project` passed instead of only the current project?

Reply:
I see from https://gist.github.com/anmolbabu/2077733486031cf6ad92205a56e6275d that the `odo list --project <proj_name>` works only for the current project immaterial of the project passed with --project